python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -53,9 +53,9 @@ class RotatorCsc(hexrotcomm.BaseCsc):
         The initial state of the CSC.
         Must be `lsst.ts.salobj.State.STANDBY` unless simulating
         (``simulation_mode != 0``).
-    settings_to_apply : `str`, optional
-        Settings to apply if ``initial_state`` is `State.DISABLED`
-        or `State.ENABLED`.
+    override : `str`, optional
+        Configuration override file to use if ``initial_state`` is
```

Comment:
Please consider changing `use` to `apply`, to be like develop (and all my other CSCs)